src/plugins/workspace/public/components/workspace_form/select_data_source_panel.tsx

```diff
@@ -130,7 +130,7 @@ export const SelectDataSourcePanel = ({
           <EuiFlexItem grow={false}>{addDirectQueryConnectionsButton}</EuiFlexItem>
         )}
       </EuiFlexGroup>
-      <EuiSpacer size="m" />
+      {assignedDataSourceConnections.length > 0 && <EuiSpacer size="m" />}
       <EuiFlexItem style={{ maxWidth: 768 }}>
```

Comment:
Can we remove this FlexItem? Seems not flex container outside.

Reply:
But it is used to limit the width.

Reply:
The max width already exists in the parent container. 
See https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/src/plugins/workspace/public/components/workspace_creator/workspace_creator_form.tsx#L79
I think we can update like below:
```tsx
{assignedDataSourceConnections.length > 0 && <><EuiSpacer size="m" />{renderTableContent()}</>}
```

Reply:
I didn't notice that, thanks